docusaurus/docs/iOS/swiftui/chat-channel-components/message-list.md

```diff
@@ -44,28 +44,28 @@ This code snippet is normally located in the `AppDelegate.swift` file inside of
-| [dateIndicatorPlacement](#dateindicatorplacement)                               | `DateIndicatorPlacement`   | `.overlay`                       |
-| [pageSize](#pagesize)                                                           | `Int`                      | `50`                             |
-| [messagePopoverEnabled](#messagepopoverenabled)                                 | `Bool`                     | `true`                           |
-| [doubleTapOverlayEnabled](#doubletapoverlayenabled)                             | `Bool`                     | `false`                          |
-| [becomesFirstResponderOnOpen](#becomesfirstresponderonopen)                     | `Bool`                     | `false`                          |
-| [maxTimeIntervalBetweenMessagesInGroup](#maxtimeintervalbetweenmessagesingroup) | `TimeInterval`             | `60`                             |
-| [cacheSizeOnChatDismiss](#cachesizeonchatdismiss)                               | `Int`                      | `1024 * 1024 * 100`              |
+| [`messageListType`](#messagelisttype)                                             | `MessageListType`          | `.messaging`                     |
+| [`typingIndicatorPlacement`](#typingindicatorplacement)                           | `TypingIndicatorPlacement` | `.bottomOverlay`                 |
+| [`groupMessages`](#groupmessages)                                                 | `Bool`                     | `true`                           |
+| [`messageDisplayOptions`](#messagedisplayoptions)                                 | `MessageDisplayOptions`    | `MessageDisplayOptions()`        |
+| [`messagePaddings`](#messagepaddings)                                             | `MessagePaddings`          | `MessagePaddings(horizontal: 8)` |
+| [`dateIndicatorPlacement`](#dateindicatorplacement)                               | `DateIndicatorPlacement`   | `.overlay`                       |
+| [`pageSize`](#pagesize)                                                           | `Int`                      | `50`                             |
+| [`messagePopoverEnabled`](#messagepopoverenabled)                                 | `Bool`                     | `true`                           |
+| [`doubleTapOverlayEnabled`](#doubletapoverlayenabled)                             | `Bool`                     | `false`                          |
+| [`becomesFirstResponderOnOpen`](#becomesfirstresponderonopen)                     | `Bool`                     | `false`                          |
+| [`maxTimeIntervalBetweenMessagesInGroup`](#maxtimeintervalbetweenmessagesingroup) | `TimeInterval`             | `60`                             |
+| [`cacheSizeOnChatDismiss`](#cachesizeonchatdismiss)                               | `Int`                      | `1024 * 1024 * 100`              |
```

Comment:
I don't think these are meant to have `` CC @martinmitrevski

Reply:
yes, these are links, we should revert these @jeroenleenarts.